modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -33,18 +33,35 @@
     private String configLocation;
     private String initScriptPath;
     private boolean enableJmxReporting;
+    private boolean authenticationEnabled = false;
+    private String userName;
+    private String password;
 
     public CassandraContainer() {
         this(IMAGE + ":3.11.2");
     }
 
+    public CassandraContainer(String userName, String password) {
```

Comment:
We prefer fluent-style configuration (`withX` methods), let's not add more constructors

Reply:
Removed all new constructors. Added 2 new overloaded methods withAuthentication() and withAuthentication(String userName, String password)